template/tsconfig/vitest/tsconfig.vitest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "@vue/tsconfig/tsconfig.node.json",
+  "include": ["src/**/__tests__/*"],
+  "compilerOptions": {
+    "composite": true,
+    "types": ["node", "jsdom"]
```

Comment:
I intentionally didn't include the Vitest types (which injects globals such as `describe`, `it`), because explicit is better than implicit.